docs/products/kafka/howto/schema-registry.rst

```diff
@@ -0,0 +1,149 @@
+
+
+Create version 1 of the Avro schema
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+To create an Avro schema, you need a definition file. As example you can use a **click record** schema defined in JSON and stored in a file named ``ClickRecord.avsc`` containing the following:
+
+::
+
+    {"type": "record",
+      "name": "ClickRecord",
+      "namespace": "io.aiven.avro.example",
+      "fields": [
+         {"name": "session_id", "type": "string"},
+         {"name": "browser", "type": ["string", "null"]},
+         {"name": "campaign", "type": ["string", "null"]},
+         {"name": "channel", "type": "string"},
+         {"name": "referrer", "type": ["string", "null"], "default": "None"},
+         {"name": "ip", "type": ["string", "null"]}
+      ]
+    }
+
+The JSON configuration above defines a schema named ``ClickRecord`` in the namespace ``io.aiven.avro.example`` with fields ``session_id``, ``browser``, ``campaign``, ``channel``, ``referrer`` and ``ip`` and related data types. 
+
+Once the schema is defined, you need to compile it, and it can be done **manually** or **automatically**.
+
+Manual schema compilation
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+In case of manual schema compilation, download ``avro-tools-1.11.0.jar`` from https://avro.apache.org/releases.html#Download or via maven using the following::
+
+    mvn org.apache.maven.plugins:maven-dependency-plugin:2.8:get -Dartifact=org.apache.avro:avro-tools:1.11.0:jar -Ddest=avro-tools-1.11.0.jar
+
+The schema defined in the previous step, can be now compiled to produce a Java class ``ClickRecord.java`` in the ``io.aiven.avro.example`` package (taken from the ``namespace`` parameter)::
+
+    java -jar avro-tools-1.11.0.jar compile schema ClickRecord.avsc .
+
+Auto schema compilation
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+With auto, the schema is compiled during the project build with, for example, ``maven-avro-plugin`` or ``gradle-avro-plugin``.
+The following is a configuration example for ``maven-avro-plugin`` when ``ClickRecord.avsc`` is stored in the path ``src/main/avro/ClickRecord.avsc``::
+
+    <plugin>
+        <groupId>org.apache.avro</groupId>
+        <artifactId>avro-maven-plugin</artifactId>
+        <version>${avro-maven-plugin.version}</version>
+        <executions>
+            <execution>
+                <id>schemas</id>
+                <phase>generate-sources</phase>
+                <goals>
+                    <goal>schema</goal>
+                    <goal>protocol</goal>
+                    <goal>idl-protocol</goal>
+                </goals>
+                <configuration>
+                    <sourceDirectory>${project.basedir}/src/main/avro/</sourceDirectory>
+                    <outputDirectory>${project.basedir}/src/main/generated-sources/</outputDirectory>
+                </configuration>
+            </execution>
+        </executions>
+    </plugin>
+
```

Comment:
And here as well, just to highlight what is going to appear in `${project.basedir}/src/main/generated-sources/` once the plugin does the generation step.

Reply:
done